datafusion/core/src/physical_plan/sorts/sort_preserving_merge.rs

```diff
@@ -269,9 +284,6 @@ pub(crate) struct SortPreservingMergeStream {
     /// The sorted input streams to merge together
     streams: MergingStreams,
 
-    /// Drop helper for tasks feeding the input [`streams`](Self::streams)
-    _drop_helper: AbortOnDropMany<()>,
```

Comment:
We no longer need this as it is handled by the individual RecordBatchReceiverStream